### DISCUSSION TITLE
Using BZAR Zeek Scripts

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#11298] [General] [answered]
Hello All!

I know that BZAR is disabled by default in the zeek config, but I was able to add bzar to the gobal.sls file using the instructions [Here](https://docs.securityonion.net/en/2.3/zeek.html?highlight=known#other-zeek-logs)

The Zeek containers are running without error and the BZAR scripts are loaded, as confirmed by catting  loaded_scripts.log.

I don't see any output files named bzar.  How would I be able to check inside Security Onion to see verify that the bzar is working?

**Comment by dougburks** [comment, marked as answer]
Most BZAR detections generate Zeek notices:
https://github.com/mitre-attack/bzar

So you might try to simulate some of the behavior listed at https://github.com/mitre-attack/bzar and then go to [SOC Dashboards](https://docs.securityonion.net/en/2.3/dashboards.html) and select the `Zeek Notice` dashboard.

**Reply by oneCrazyAdmin** [reply]
Thank you!  It was not really clear in the documentation that the bzar writes to the Zeek Notice.

**Reply by dougburks** [reply]
I've updated the documentation.
